js/dashboard.js

```diff
@@ -50,142 +42,139 @@ const dashboardScreen = (meetings) => {
 
-// // Delete Slot
-// const deletSlot = (roomId, deleteSlotId) => {
-//     if (confirm("Are you sure you want to delete this slot?")) {
-//         let rooms = JSON.parse(localStorage.getItem('rooms'));
-//         const roomIndex = rooms.findIndex((room) => Number(room.roomId) === roomId);
-//         rooms[roomIndex].roomSlots.findIndex((slots)=> slots.roomSlotId === deleteSlotId)
-//         if (roomIndex !== -1) {
-//             rooms[roomIndex].roomSlots.splice(roomIndex, 1);
-//             localStorage.setItem("rooms", JSON.stringify(rooms));
-//             alert("Slot deleted successfully");
-//         } else {
-//             alert("Slot not found");
-//         }
-//         location.reload();
-//     }
-// }
+  document.getElementById("editModalMeetingTitle").value = user.meeting_name;
+  document.getElementById("editModalRoom").value = user.fk_room_id;
+  document.getElementById("editModalDate").value = user.meeting_date.slice(
+    0,
+    10
+  );
+  document.getElementById("editModalClockInTime").value = user.start_time;
+  document.getElementById("editModalClockOutTime").value = user.end_time;
+};
 
-// // delete Room
-// const deleteRoom = (roomId) => {
-//     localStorage.setItem("roomId", roomId);
-//     if (confirm("Are you sure you want to delete this room?")) {
-//         let rooms = JSON.parse(localStorage.getItem('rooms'));
-//         const roomIndex = rooms.findIndex((room) => Number(room.roomId) === roomId);
-//         if (roomIndex !== -1) {
-//             rooms.splice(roomIndex, 1);
-//             localStorage.setItem("rooms", JSON.stringify(rooms));
-//             alert("Room deleted successfully");
-//         } else {
-//             alert("Room not found");
-//         }
-//         location.reload();
-//     }
-// }
+// Edit Meeting
+const updateMeeting = () => {
+  let meeting_name = document.getElementById("editModalMeetingTitle").value;
+  let fk_room_id = document.getElementById("editModalRoom").value;
+  let meeting_date = document.getElementById("editModalDate").value;
+  let start_time = document.getElementById("editModalClockInTime").value;
+  let end_time = document.getElementById("editModalClockOutTime").value;
+  let updated_by = 2;
 
-// // for calender
-// $(document).ready(function () {
-//     // Initialize the date picker
-//     $("#roomSlotDate").datepicker({
-//       dateFormat: "dd-mm-yy", // Set the date format to yyyy-mm-dd
-//       minDate: 0, // Set the minimum selectable date as today
-//     });
-//     // Clear the date picker field
-//     $("#roomSlotDate").val("");
-//   });
+  if (
+    !meeting_name &&
+    !fk_room_id &&
+    !meeting_date &&
+    !start_time &&
+    !end_time
+  ) {
+    alert("Please, Fill all Field");
+  } else {
+    const id = localStorage.getItem("updateMeetingId");
+    axios
+      .put(`http://localhost:8080/meeting/update/${id}`, {
+        meeting_name,
+        fk_room_id,
```

Comment:
create variable and assign the variable in API param

Reply:
done